app/src/utils/utils.ts

```diff
@@ -28,22 +42,69 @@ export function isValidAddress(val: string | undefined) {
   return val && isAddress(val);
 }
 
-// Expects a unix timestamp and will return a human readable message of how far in the past/future it is
-export function daysAgo(val = 0) {
-  // Use a formatter to establish "in 10 days" vs "10 days ago"
-  const formatter = new Intl.RelativeTimeFormat();
-  // Number of days since now
-  const deltaDays = (val * 1000 - Date.now()) / (1000 * 3600 * 24);
+// --- Grants + Cart ---
+const CART_KEY = 'cart';
+const DEFAULT_CONTRIBUTION_TOKEN_ADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F'; // DAI
```

Comment:
should we use from token.dai.address?

Reply:
That `token` mapping from symbol to token info only exists in the `contracts` package, and I don't see a need for mapping symbol to token in the app directory, so my opinion is that we can keep this as is.

Reply:
lgtm